torch/csrc/jit/passes/quantization.cpp

```diff
@@ -135,6 +135,7 @@ bool isFunctionNode(Node* n,
 
 bool nodeQuantizable(Node* n) {
   return isFunctionNode(
+      n,
```

Comment:
Looks like this is supposed to be part of the previous diff.  Please split/merge to make sure bisecting works.